CriteoPublisherSdk/Sources/Standalone/CRBannerView.m

```diff
@@ -113,12 +111,7 @@ - (void)loadAd {
 
   if ([bid isEmpty]) return [self safelyNotifyAdLoadFail:CRErrorCodeNoFill];
 
-  if (!bid.displayUrl)
-    return [self safelyNotifyAdLoadFail:CRErrorCodeInternalError
-                            description:@"No display URL in bid response"];
-
-  [self dispatchDidReceiveAdDelegate];
-  [self loadWebViewWithDisplayUrl:bid.displayUrl];
+  [self loadAdWithDisplayData:bid.displayUrl];
```

Comment:
Regarding the naming I went with `displayAd` on Android because this is what the method effectively does. I would say that `loadAd` has the semantic of fetching bids from CDB.

Reply:
Done